geo/src/algorithm/convert_angle_unit.rs

```diff
@@ -0,0 +1,83 @@
+use geo_types::Coord;
+use geo_types::CoordFloat;
+
+use crate::{MapCoords, MapCoordsInPlace};
+
+pub trait ToRadians<T: CoordFloat>: Sized + MapCoords<T, T, Output = Self> {
+    fn to_radians(&self) -> Self {
+        self.map_coords(|Coord { x, y }| Coord {
+            x: x.to_radians(),
+            y: y.to_radians(),
+        })
+    }
+}
+impl<T: CoordFloat, G: MapCoords<T, T, Output = Self>> ToRadians<T> for G {}
+
+pub trait ToRadiansInPlace<T: CoordFloat>: MapCoordsInPlace<T> {
+    fn to_radians_in_place(&mut self) {
+        self.map_coords_in_place(|Coord { x, y }| Coord {
+            x: x.to_radians(),
+            y: y.to_radians(),
+        })
+    }
+}
+impl<T: CoordFloat, G: MapCoordsInPlace<T>> ToRadiansInPlace<T> for G {}
+
+pub trait ToDegrees<T: CoordFloat>: Sized + MapCoords<T, T, Output = Self> {
+    fn to_degrees(&self) -> Self {
+        self.map_coords(|Coord { x, y }| Coord {
+            x: x.to_degrees(),
+            y: y.to_degrees(),
+        })
+    }
+}
+impl<T: CoordFloat, G: MapCoords<T, T, Output = Self>> ToDegrees<T> for G {}
+
+pub trait ToDegreesInPlace<T: CoordFloat>: MapCoordsInPlace<T> {
+    fn to_degrees_in_place(&mut self) {
+        self.map_coords_in_place(|Coord { x, y }| Coord {
+            x: x.to_degrees(),
+            y: y.to_degrees(),
+        })
+    }
+}
+impl<T: CoordFloat, G: MapCoordsInPlace<T>> ToDegreesInPlace<T> for G {}
+
+#[cfg(test)]
+mod tests {
+    use std::f64::consts::PI;
+
+    use approx::assert_relative_eq;
+    use geo_types::Line;
+
+    use super::*;
+
+    lazy_static::lazy_static! {
```

Comment:
Rather than adding this dependency, can you just turn these into functions that return the `Line`s? Just having `lazy_static` used in this one spot doesn't seem worth the addition of a dependency

Reply:
Fair, resolved 🤗 Using lazy statics for mocking is a "design pattern" I've gotten used to liking. Incorporating that pattern should be probably be done in all tests where it would be appropriate, and in separate (low-priority) issue+pr.